torch/lib/c10d/reducer.cpp

```diff
@@ -1559,6 +1559,14 @@ void Reducer::record_backward_comm_end_time() {
   }
 }
 
+void Reducer::set_static_graph() {
+  TORCH_INTERNAL_ASSERT(
```

Comment:
nit: This should be `TORCH_CHECK` since it is a user error and not an internal error.